recipes/_configure_server.rb

```diff
@@ -41,8 +41,11 @@
 # default sslPassword value is 'password'
 server_stanzas['sslConfig']['sslPassword'] = CernerSplunk::ConfTemplate.compose encrypt_noxor_password, CernerSplunk::ConfTemplate::Value.constant(value: 'password')
 
-# Indexer Cluster Configuration
 case node['splunk']['node_type']
+when :forwarder
+  # Apparently there is no option to configure site awareness for different mutlisite clusters.
+  # https://answers.splunk.com/answers/590556/how-to-configure-site-awareness-on-forwarders-for.html
+  server_stanzas['general']['site'] = node['splunk']['forwarder_site']
```

Comment:
This attribute will be set even if it is not a multisite cluster. I tested and the forwarder is able to forward to single site cluster with this attribute set to site0.